components/searchable_channel_list.jsx

```diff
@@ -276,6 +213,18 @@ export default class SearchableChannelList extends React.PureComponent {
             );
         }
 
+        const Item = ({index, style}) => {
+            let content;
+
+            if (this.isItemLoaded(index)) {
+                content = this.createChannelRow(this.props.channels[index]);
+            } else {
+                content = 'Loading...';
```

Comment:
This should be wrapped in a `FormattedMessage` so that it'll be translatable. You can add a `more_channels.loading` string for it

Reply:
okay updated 👍 

Didn't have time to write e2e tests.